### ISSUE TITLE
no scriptlets

### ISSUE THREAD

**Issue by ghost** [FedoraUnited/deadbeef#2]
_deadbeef.spec no scriptlets icon cache, desktop-database_

_https://fedoraproject.org/wiki/Packaging:Scriptlets#Icon_Cache_
_https://fedoraproject.org/wiki/Packaging:Scriptlets#desktop-database_

**Comment by ghost** [comment]
ping?

**Comment by paulcarroty** [comment]
@kuboosoft pls merge scriplets from first commit.

**Comment by ghost** [comment]
_Need to add in deadbeef.spec_

```
%post
/usr/bin/update-desktop-database &> /dev/null || :
/bin/touch --no-create %{_datadir}/icons/hicolor &> /dev/null || :

%postun
/usr/bin/update-desktop-database &> /dev/null || :
if [ $1 -eq 0 ] ; then
    /bin/touch --no-create %{_datadir}/icons/hicolor &> /dev/null || :
    /usr/bin/gtk-update-icon-cache %{_datadir}/icons/hicolor &> /dev/null || :
fi

%posttrans
/usr/bin/gtk-update-icon-cache %{_datadir}/icons/hicolor &> /dev/null || :
```

**Comment by paulcarroty** [comment]
fixed in 0.7.2-2
